dev_tools/scripts/gdb-ginkgo.py

```diff
@@ -220,12 +225,28 @@ def display_hint(self):
         return 'array'
 
 
+class GkoHalfPrinter:
+    "Print a gko::half"
+
+    def __init__(self, val):
+        # GDB doesn't seem to consider the user-defined conversion in its Value.cast,
+        # so we need to call the conversion operator explicitly
+        address = hex(val.address)
+        self.float_val = gdb.parse_and_eval(f"reinterpret_cast<gko::half*>({address})->operator float()")
```

Comment:
If that function is ever not exported, we may need to implement an xfunction for it

Reply:
I am not familar with that. This is done by @MarcelKoch  
could you implement that?

Reply:
We likely don't need it now, just for future reference